public/jmap/map.d.ts

```diff
@@ -193,6 +198,14 @@ declare interface JBasemap {
   maxzoom?: number
   tileSize?: number
   extent?: JBoundaryBox
+  attributions? : JMapAttribution[]
+}
+
+declare interface JMapAttribution {
+  id: string
+  text: string | undefined
```

Comment:
text: string | undefined // text and/or imageUrl param(s) must be set
imageUrl: string | undefined // imageUrl and/or text param(s) must be set